src/Diagnostics.RuntimeHost/appsettings.json

```diff
@@ -104,5 +104,9 @@
     "ClientId": "",
     "AllowedAppIds": ""
   },
-  "CloudDomain": "PublicAzure"
+  "CloudDomain": "PublicAzure",
+  "OutboundConnectivity": {
+    "IsCheckEnabled": true,
+    "PingUrl": "http://google.com/generate_204"
```

Comment:
Can the URL just be one of our internal systems? I don't think this will fly in some other cloud environments.

Reply:
The url is configurable. It has to outside our own app ecosystem because if any of our internal system is down, we don't want runtime host to think it has lost network connectivity. So we use an external URL. For other clouds the check can be disabled and later enabled when we have the correct URL to use.